spec/features/courses/requirements_spec.rb

```diff
@@ -1,21 +1,26 @@
 require 'rails_helper'
 
 feature 'Course requirements', type: :feature do
-  let(:course_jsonapi) do
-    jsonapi(
-      :course,
-      provider: jsonapi(:provider, provider_code: 'A0')
-    )
+  let(:course_1) { jsonapi :course, name: 'English', provider: provider, include_nulls: [:accrediting_provider] }
+  let(:course_2) { jsonapi :course, name: 'Biology', include_nulls: [:accrediting_provider] }
+  let(:courses)  { [course_2] }
```

Comment:
I know this has changed since my previous comment, but still ... why the need for the intermediary variable `courses`?

Reply:
Removed 👍